openapi/src/parts/workspace.yaml

```diff
@@ -619,3 +647,52 @@ components:
       items:
         uniqueItems: true
         type: string
+
+    WsmPolicyDepth:
+      type: integer
+      description: Explain source depth.
+      minimum: 0
+      default: 1
+
+    WsmPolicyExplanation:
```

Comment:
I think we should provide more information to the WSM client than what TPS hands out. See the design doc for what I think should be returned: [WSM REST API](https://docs.google.com/document/d/1KRQ57LmJ-TS_MHxYuStATVxTp2rwz4LZbqwPODJBs6I/edit#heading=h.7y616an1qwza)

Reply:
done.